src/client/components/Delete_Comfim/ModalList.component.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Button from '../button/button.component';
+
+const ModalList = ({ isShowing, hide, onClick }) => {
```

Comment:
- You don't really need to pass `isShowing` (see the comment below)
- You don't need to pass `hide` because you can pass `setIsShowing` instead, and then replace it with `() => setIsShowing(false)`